core/config/env/env.go

```diff
@@ -0,0 +1,52 @@
+package env
+
+import (
+	"os"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/v2/core/store/models"
+)
+
+var (
+	Config = Var("CL_CONFIG")
+
+	// LOOPP commands and vars
+	MedianPluginCmd   = Var("CL_MEDIAN_CMD")
+	SolanaPluginCmd   = Var("CL_SOLANA_CMD")
+	StarknetPluginCmd = Var("CL_STARKNET_CMD")
+	// PrometheusDiscoveryHostName is the externally accessible hostname
+	// published by the node in the `/discovery` endpoint. Generally, it is expected to match
+	// the public hostname of node.
+	// Cluster step up like kubernetes may need to set this explicitly to ensure
+	// that Prometheus can discovery LOOPps.
+	// In house we observed that the resolved value of os.Hostname was not accessible to
+	// outside of the given pod
+	PrometheusDiscoveryHostName = Var("CL_PROMETHEUS_DISCOVERY_HOSTNAME")
+	// EnvLooopHostName is the hostname used for HTTP communication between the
+	// node and LOOPps. In most cases this does not need to be set explicitly.
+	LooppHostName = Var("CL_LOOPP_HOSTNAME")
```

Comment:
pre-existing, but since you are touching it:
```suggestion
	LOOPPHostName = Var("CL_LOOPP_HOSTNAME")
```